draft-timbru-sidrops-publication-server-bcp-00.md

```diff
@@ -182,14 +182,15 @@ In this section, we will elaborate on the following recommendations:
 
 ## Consistent Content
 
-A naive implementation of the Rsync Repository can change the repository
-content while RPs transfer files. Even when the repository is consistent from
-the repository server's point of view, clients may read an inconsistent set of
-files. Clients may get a combination of newer and older files. This "phantom
-read" can lead to unpredictable and unreliable results. While modern RPs will
-treat such inconsistencies as a "Failed Fetch" ([@!RFC9286]), it is best to
-avoid this situation since a failed fetch for one repository can cause the
-rejection of the publication point for a sub-CA when resources change.
+A naive implementation of the Rsync Repository might change the repository
+content mid-flight while RPs transfer files. Even when the repository is
+consistent from the repository server's point of view, clients may read an
+inconsistent set of files. Clients may get a combination of newer and older
+files. This "phantom read" can lead to unpredictable and unreliable results.
+While modern RPs will treat such inconsistencies as a "Failed Fetch"
+([@!RFC9286]), it is best to avoid this situation since a failed fetch for one
+repository can cause the rejection of the publication point for a sub-CA when
+resources change.
```

Comment:
Missed this beforehand. My understanding (and see discord discussion) is that on failed fetch, anything below is not re-validated and cached results are used.

Reply:
@job what is your read of that section  in manifest-bis?

Reply:
TL;DR: I am not sure that we should go into much detail here. Better refer to RFC 9286 section 6.6?

My understanding was that cached objects should be used - but that if those turn out to be invalid as well - then the publication point may be rejected.

I am not entirely sure why the 'resources changed' reference is here. It's a corner case that requires more explanation and this doc may not be the place to do it.

The problem is that if a cached publication point is used with a delegated CA certificate for a sub-CA. And the resources for that CA were extended, but this is not visible to the RP (because they use cached), then the child may appear to be overclaiming.

This situation is less likely to occur if both CAs publish in the same repo (e.g. hosted CAs in the RIR service).

This situation can also occur naturally - without failed fetches - because the child may request an extended certificate using RFC 6492, and start to use new resources before the paren has published the new certificate and/or RPs may simply see the child repo updates before the parent CA's repo updates. This should be addressed in 8181/6492++.